src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Extensions/DbContextTestExtensions.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+
+public static class DbContextTestExtensions
+{
+    public static async Task ClearTestData<TDbContext>(this TDbContext context) where TDbContext : DbContext
+    {
+        if (context.Database.IsNpgsql())
+        {
+            var tables = context.Model.GetEntityTypes()
+                .Select(type => type.GetTableName())
+                .Distinct()
+                .ToList();
```

Comment:
`GetTableName` can return null her.  In some places where this code has been moved from we've handled this with `.Cast<string>()` so you could add that in here?

Reply:
Would suggest using `OfType<string>()` instead. When using `Cast`, if the type is incorrect it can throw an exception, whereas `OfType` just filters unexpected types out

Reply:
Great, thanks @ntsim.

Reply:
Cool added that, thanks @benoutram